src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/arm/eltwise.cpp

```diff
@@ -0,0 +1,109 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/classes/eltwise.hpp"
+#include "shared_test_classes/single_layer/eltwise.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+#include "test_utils/fusing_test_utils.hpp"
+
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+using namespace ngraph::helpers;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+namespace Eltwise {
+namespace {
+static const std::vector<InputShape> bitwise_in_shapes_4D = {
+    {
+        {1, -1, -1, -1},
+        {
+            {1, 3, 4, 4},
+            {1, 3, 1, 1},
+            {1, 1, 1, 1}
+        }
+    },
+    {{1, 3, 4, 4}, {{1, 3, 4, 4}}}
```

Comment:
Can we also cover shapes with bigger size than simd_w?

Reply:
done, thanks